frontend/pages/wiki/_page.vue

```diff
@@ -29,18 +27,12 @@ export default {
       }
     ])
     const wikiApiUrl = `${this.$config.wikibaseApiUrl}?action=parse&page=${this.wikiPage}&prop=wikitext&formatversion=2&format=json&origin=*`
-    this.contentToView = await this.$wikibase.wbFetcher(wikiApiUrl)
+    const html = await this.$wikibase.wbFetcher(wikiApiUrl)
       .then(data => this.contentPage(data))
+    this.contentToView = this.$sanitize(html)
   },
 
   methods: {
-    removeLocalePrefix (page) {
```

Comment:
why are you reverting this change?

Reply:
let me review, there's some version difference i think here